styleguide/source/_patterns/00-atoms/00-intro/_layout-intro-styling.scss

```diff
@@ -20,5 +20,5 @@
 .layout-intro-styling blockquote code {
   display: block;
   border: 0;
-  padding: .5em;
-}
\ No newline at end of file
+  padding: 0.5em;
```

Comment:
Suggestion to remove leading zero since I think the linter prefers to strip them out